src/applications/netarkivet/apply_filters.py

```diff
@@ -0,0 +1,136 @@
+"""
+Apply deduplication and quality filter to NetArkivet Text Corpus (NAT).
+
+Dependent on:
+    src/applications/netarkivet/add_duplicate_column.py
+
+Authors:
+    Kenneth Enevoldsen
+"""
+from typing import Iterable, Optional, List, Union
+from pathlib import Path
+import glob
+import random
+from contextlib import ExitStack
+import os
+
+import ndjson
+from wasabi import msg
+
+
+def shuffle_buffer(x: Iterable, buffer_size: int) -> Iterable:
+    """Creates shuffle buffer from iterable.
+    Args:
+        x (Iterable): An iterable you want shuffled
+        buffer_size (int): The buffer shuffle
+    Returns:
+        Iterable: An iterable which is shuffled using a shuffle buffer
+    Example:
+        >>> shuffled = shuffle_buffer(x =[1,2, 3, 4, 5], buffer_size=2)
+        >>> print(list(shuffled))
+        [2, 1, 4, 3, 5]
+    """
```

Comment:
If you want to do Google-style docstrings you should include a blank line between the headings:
```suggestion
    """Creates shuffle buffer from iterable.
    
    Args:
        x (Iterable): An iterable you want shuffled
        buffer_size (int): The buffer shuffle
    
    Returns:
        Iterable: An iterable which is shuffled using a shuffle buffer
    
    Example:
        >>> shuffled = shuffle_buffer(x =[1,2, 3, 4, 5], buffer_size=2)
        >>> print(list(shuffled))
        [2, 1, 4, 3, 5]
    """
```